ant-cli/tests/single_file_download_fix.rs

```diff
@@ -0,0 +1,50 @@
+// Test for AUTO-177: Verify single public file doesn't get downloaded twice
+// This test ensures that when a file fails to deserialize as PublicArchive,
+// we use the already-downloaded bytes instead of re-downloading
+
+use autonomi::{files::PublicArchive, Bytes};
+
+#[test]
```

Comment:
the newly created two tests are just unit test for serialize/deserialize, and outlined a testing setup ?

for a the fix to be fully verified, the test shall work against a local testnet, and being an E2E test involving upload/download and verify the fetched content.